<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit</class>
 <widget class="QDialog" name="credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="input_layout">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #E2E4E6;
color: black;

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>211</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Loan amount:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="amount_val">
       <property name="styleSheet">
        <string notr="true">background-color: white;
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Term</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="term_val">
       <property name="styleSheet">
        <string notr="true">background-color: white;
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>139</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="months">
       <property name="text">
        <string>months</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="years">
       <property name="text">
        <string>years</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>131</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Interest rate per year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="interest_rate">
       <property name="styleSheet">
        <string notr="true">background-color: white;
text-align: right;
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>185</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="annuity">
       <property name="text">
        <string>annuity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="diff">
       <property name="text">
        <string>differentiated</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;</string>
    </property>
    <property name="text">
     <string>Loan calculator</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="calc_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: #000;
border-bottom: 1px solid #b5b5b5;
border-radius:  5px;
background-color: #fff;
}

QPushButton::pressed {
background-color: grey;
}</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>208</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>171</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F7CFCA;
color: black;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="overpayment">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">qproperty-alignment: AlignCenter;</string>
       </property>
       <property name="text">
        <string>Overpayment on credit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="overpayment_val">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
qproperty-alignment: AlignCenter;
}</string>
       </property>
       <property name="text">
        <string>00.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="total_payments">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">qproperty-alignment: AlignCenter;</string>
       </property>
       <property name="text">
        <string>Total payment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="total_payment_val">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
qproperty-alignment: AlignCenter;
}</string>
       </property>
       <property name="text">
        <string>00.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="monthly_payments">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">qproperty-alignment: AlignCenter;</string>
       </property>
       <property name="text">
        <string>Monthly payments</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="monthly_payment_val">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
qproperty-alignment: AlignCenter;
}</string>
       </property>
       <property name="text">
        <string>00.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
